.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm test
```

Comment:
It looks like you are not using `lint-staged` at all. Any command added in the `.husky/pre-commit` file will run whenever someone tries to commit, but instead of running tests (which we don’t have in this project), you should run `lint-staged`.  

`lint-staged` ensures that linting and formatting are only applied to the files being committed, making the pre-commit process more efficient. This prevents unnecessary processing of unchanged files and speeds up commits.  

You can refer to this article for a guide on setting up Husky with `lint-staged`:  
[Getting Started with Husky and Lint-Staged](https://duncanlew.medium.com/getting-started-with-husky-and-lint-staged-for-pre-commit-hooks-c2764d8c9ae)